include/libjsonnet.h

```diff
@@ -24,8 +24,6 @@ limitations under the License.
  * of using the library.
  */
 
-#define LIB_JSONNET_VERSION "v0.12.1"
-
 /** Return the version string of the Jsonnet interpreter.  Conforms to semantic versioning
  * http://semver.org/ If this does not match LIB_JSONNET_VERSION then there is a mismatch between
```

Comment:
The idea was that you would be able to compare the two to check the .so was correct, but that may not be particularly useful, because we basically have to be backwards compatible anyway.  I don't mind removing it, but this comment needs updating. Also setup.py expects to find the version in this specific file.